pom.xml

```diff
@@ -11,7 +11,7 @@
     <packaging>jar</packaging>
     <groupId>org.webjars</groupId>
     <artifactId>jstree</artifactId>
-    <version>3.3.8-SNAPSHOT</version>
+    <version>3.3.8</version>
```

Comment:
This needs to be `3.3.8-SNAPSHOT` since Maven does that actual release.

Reply:
Done! thanks for advice.